packages/google_maps_flutter/google_maps_flutter_android/android/src/test/java/io/flutter/plugins/googlemaps/ConvertTest.java

```diff
@@ -26,4 +31,54 @@ public void ConvertToPointsConvertsThePointsWithFullPrecision() {
     Assert.assertEquals(latitude, latLng.latitude, 1e-15);
     Assert.assertEquals(longitude, latLng.longitude, 1e-15);
   }
+
+  @Test
+  public void ConvertClustersToJsonReturnsCorrectData() {
+    String clusterManagerId = "cm_1";
+    StaticCluster<MarkerBuilder> cluster = new StaticCluster<>(new LatLng(43.00, -87.90));
```

Comment:
This test could be easier to read if you had position1/2/3 defined a the top then passed those positions into the constructor and compared to those positions in the asserts.